module-1/src/Components/MovieDeleteForm/MovieDeleteForm.tsx

```diff
@@ -0,0 +1,67 @@
+import { toast } from "react-toastify";
+
+interface MovieDeleteFormProps {
+  title: string;
+  movie?: Movie;
+  closeCallback: () => void;
+}
+export const MovieDeleteForm = ({
+  title,
+  movie,
+  closeCallback,
+}: MovieDeleteFormProps) => {
+  const handleClose = () => closeCallback();
+
+  const deleteMovie = () => {
+    let url = `http://localhost:4000/movies/${movie?.id}`;
+
+    fetch(url, {
+      method: "DELETE",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+      },
+    })
+      .then((response) => {
+        if (response.ok) {
+          toast.success("Congratulations!\nMovie Deleted succesfully.", {
+            autoClose: 2000,
+            onClose: () => handleClose(),
+          });
+        } else {
+          return Promise.reject(response);
+        }
+      })
+      .catch(async (response) => {
+        let str = await response.json();
+        console.log(str);
+        toast.error(`Something went wrong.`);
+      });
+  };
+
+  return (
+    <div className="mx-auto">
+      <h1 className="text-white font-Montserrat text-center m-5">{title}</h1>
+      <p className="text-white font-Montserrat text-center m-5">
+        Are you sure you want to Delete this Movie?
+      </p>
+
+      <div className="flex justify-end mx-3 my-1">
+        <button
+          type="button"
+          onClick={() => handleClose()}
+          className="bg-custom_black hover:bg-slate-900 text-custom_pink  border border-custom_pink rounded-md ml-2 py-1 px-2"
+        >
+          Cancel
+        </button>
+        <button
+          type="submit"
```

Comment:
Button type `submit` makes sense in the context of a `form`. Either make it a proper html form or use type `button`.

Reply:
fixed